app/screens/navigation.ts

```diff
@@ -449,6 +450,11 @@ export function goToScreen(name: AvailableScreens, title: string, passProps = {}
     const theme = getThemeFromState();
     const isDark = tinyColor(theme.sidebarBg).isDark();
     const componentId = NavigationStore.getVisibleScreen();
+    if (!componentId) {
+        logError('Trying to go to screen without any screen on the navigation store');
+        return '';
+    }
```

Comment:
What is the reason for this?

Reply:
In case we get here without the navigation being ready, mainly on the scenario of the other comment.

Reply:
If I remember correctly somewhere around this flow we wait for the home screen to be in the stack, perhaps we should move that here instead?